readme.md

```diff
@@ -48,12 +57,12 @@ Emoji to modify.
 Type: `number`<br>
 Values:
 
-- `0` None
-- `1` 🏻 White        *(Fitzpatrick Type-1–2)*
-- `2` 🏼 Cream white  *(Fitzpatrick Type-3)*
-- `3` 🏽 Light brown  *(Fitzpatrick Type-4)*
-- `4` 🏾 Brown        *(Fitzpatrick Type-5)*
-- `5` 🏿 Dark brown   *(Fitzpatrick Type-6)*
+- `skinTone.NONE` / `0`: None
+- `skinTone.WHITE` / `1`: 🏻 White        *(Fitzpatrick Type-1–2)*
+- `skinTone.CREAM_WHITE` / `2`: 🏼 Cream white  *(Fitzpatrick Type-3)*
+- `skinTone.LIGHT_BROWN` / `3`: 🏽 Light brown  *(Fitzpatrick Type-4)*
+- `skinTone.BROWN` / `4`: 🏾 Brown        *(Fitzpatrick Type-5)*
+- `skinTone.DARK_BROWN` / `5`: 🏿 Dark brown   *(Fitzpatrick Type-6)*
```

Comment:
No need for the `White` etc description since the constants are self-describing.